qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test-local.properties

```diff
@@ -1,10 +1,10 @@
-webapp.url=https://qa-refapp.openmrs.org/openmrs
+webapp.url=http://localhost:8080/openmrs
 login.username=admin
 login.password=Admin123
 login.location=Pharmacy
 webdriver=firefox
 login.auto=false
 headless=true
-#db.host=openmrs
+db.host=openmrs
```

Comment:
Is this also needed as part of the commit

Reply:
the db server need to be set when running on local host.